### DISCUSSION TITLE
Led indicators on G14 (2022)

### DISCUSSION THREAD

**Discussion by thduc87** [seerge/g-helper#363] [Q&A]
Hi, i'm curious if we can control the way the led indicators working like on/off, flashing, etc with g-helper? thanks in advance

**Comment by seerge** [comment]
@thduc87 i was checking existing linux asus control tools  / other open source apps, and looks like it's not possible. I would also want to disable at least annoying SSD light ....

**Reply by thduc87** [reply]
yeah too bad we can't tweak the led, thanks for your response and keep up the good work, really appreciated.
